app/src/main/java/com/zeapo/pwdstore/Migrations.kt

```diff
@@ -87,8 +87,8 @@ private fun migrateToGitUrlBasedConfig(context: Context) {
 }
 
 private fun migrateToHideAll(context: Context) {
+    context.sharedPrefs.all[PreferenceKeys.SHOW_HIDDEN_FOLDERS] ?: return
     val isHidden = context.sharedPrefs.getBoolean(PreferenceKeys.SHOW_HIDDEN_FOLDERS, false)
```

Comment:
I think that even after the commit, the key is not deleted if it was set to `false`.
How about (untested):
```suggestion
    val isHidden = context.sharedPrefs.getBoolean(PreferenceKeys.SHOW_HIDDEN_FOLDERS, null) ?: return
```

Reply:
You can't pass `null` in a `getBoolean` call, this was my first idea as well.

Reply:
I updated the tests to assert that the key's being dropped and they pass locally for me.

Reply:
> I updated the tests to assert that the key's being dropped and they pass locally for me.

Nvm, I was mistaken there.